app/packs/src/utilities/MathUtils.js

```diff
@@ -56,9 +56,42 @@ const formatBytes = (bytes, decimals = 2) => {
   return `${parseFloat((bytes / (k ** i)).toFixed(dm))} ${sizes[i]}`;
 };
 
+function parseNumericString(numberString) {
+  if (typeof numberString !== 'string') {
+    return NaN;
+  }
```

Comment:
It could be confusing, that you pass in a valid number (which is not a string), but the function returns `NaN`. 
This a discussion about expectation and could be resolved having a doc-string. 

But, what if every type is accepted and cast to a `string`? Alternatively, if a number is passed, this number could return immediately. 

I'm not sure what the best way to handle this is either and your implementation might be the best JS solution. This is just my thoughts on this.

Reply:
I feel like the function name indicates that it expects a string? 
Together with the type signature form 59e2d74, that's even more clear.
I'd find it more confusing if it *would* accept an argument of type `number`.

Reply:
Yes, you are right - i agree. Actually, I was not sure about this either. So lets keep this!